src/admin-views/widgets/components/dropdown.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Admin View: Widget Dropdown Component
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/admin-views/widgets/components/dropdown.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    http://m.tri.be/1aiy
+ *
+ * @var string  $label   Label for the checkbox.
+ * @var string  $value   Value for the checkbox.
+ * @var string  $id      ID of the checkbox.
+ * @var string  $name    Name attribute for the checkbox.
+ * @var boolean $checked Whether the checkbox is checked or not.
+ *
+ * @version TBD
+ *
+ */
+
```

Comment:
:no_entry_sign: Additional blank lines found at end of doc comment

<sub>hash: d1a4418ff656c3a46066d3585f9ceed5</sub>